### PR TITLE
Math Magicians: setup

### PR DESCRIPTION
**Pull request by normainobary2021**
In this PR, I did the following:
- Installed React application using npx create-react-app command.
- Installed and set up linters for react/redux.
- Used Gitflow principle.
- Checked for linter errors and fixed them.
- Changed some text on the home page.
- Made the README file to match the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
```
Comment (sja-thedude) [inline package.json]: - [ ] Kindly fix the error which is displayed when the app starts running.

https://user-images.githubusercontent.com/70269507/198075362-f6ed5fd2-4d72-448c-a1b5-74bff0784b54.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @normainobary2021, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] Check the comments under the review

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (normainobary2021) [reply package.json]: Okay @sja-thedude I will do that now. Thank you

**Review by topeogunleye — APPROVED** [review]
### Hi @normainobary2021, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description summary :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by normainobary2021** [comment]
Thank you @topeogunleye for the quick review

**Comment by topeogunleye** [comment]
> Thank you @topeogunleye for the quick review

@normainobary2021, you're welcome

**Pull request merged**